api/policy.go

```diff
@@ -142,6 +155,43 @@ func UnmarshalBackupPlanPolicy(payload string) (*BackupPlanPolicy, error) {
 	return &bp, nil
 }
 
+// WindowType specify the window type
+type WindowType int
+
+const (
+	WindowTypeAllow WindowType = iota
+	WindowTypeDeny
```

Comment:
Need to explain the priority, what if there are 2 entries. One is 10:00 ~ 11:00 DENY, the other is 10:00 ~ 11:00 ALLOW.